### ISSUE TITLE
"A solution"

### ISSUE THREAD

**Issue by yormi** [ed-ilyin/elmish-polymer#1]
Hi there,

Found your post on gitter while looking for the same problem you seem to have. I wanted to quickly share in case it'd be useful to you or other people.

I managed to properly compile by modifying `create-elm-app` (which I assume you use as well !?).

1. I followed the instruction in this [StackOverflow answer](https://stackoverflow.com/a/43803066/3687661).
1. The new `uglifyjs` version needs a change in the webpack config file: Wrapping the UglifyJS options like so `uglifyOptions: {compress: {...}, output: {...}}` in `config/webpack-config.prod.js`

Basically, the problem is caused by `uglify` not handling ES6 properly hence the need to use `uglify-es`.

Hope it helps !

Let me know if you have questions :)
